packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -279,7 +279,7 @@ export class ProductAttributeValue extends PureComponent {
               block="ProductAttributeValue"
               elem="SubLabel"
             >
-                { `(${subLabel})` }
+                { subLabel ? `(${subLabel})` : undefined }
             </strong>
         </span>
```

Comment:
Move it to a function where you will be rendering subLabel for checkbox label. Out there just check if subLabel is undefined then return null; If something then return Strong tag with content.